e2e/benchmark_test.go

```diff
@@ -12,8 +12,9 @@ import (
 
 // BenchmarkPutsWithSecondary  ... Takes in an async worker count and profiles blob insertions using
 // constant blob sizes in parallel
+// TODO: test variations with V2
 func BenchmarkPutsWithSecondary(b *testing.B) {
-	testCfg := TestConfig(true)
+	testCfg := TestConfig(true, false)
```

Comment:
no need to refactor in this PR but going forard the functional options pattern makes things a lot more readable. here we just see (true, false) and it's impossible to know syntactically (without an IDE to click-through) what these mean.